### ISSUE TITLE
Outlook Signature Freeze

### ISSUE THREAD

**Issue by Nerlant** [nvaccess/nvda#18109]
### Short description of the issue
When using certain signatures in Outlook in combination with NVDA, Outlook freezes when typing a new message, especially when entering a newline.

### What we tested so far
The issue initially occurred at a client's system. After narrowing down the issue to the client's signature, we conducted internal testing.
We were able to reproduce the issue on 3 out 4 tested systems. The PC not having the issue is identical to one of the PCs having the issue.
We compared NVDA, Windows and Outlook versions, all being identical as well. So currently we do not know why one PC does not have the issue.
We tried to debug the issue by modifying the NVDA UIA and Windows Event handling, hoping to find an event that is responsible to trigger the issue.
The only reliable way we found to "fix" this issue is to completely disable UI Automation and Windows event handling inside NVDA while the “New E-Mail” window of Outlook is opened.

### Steps to reproduce:

1. Save the following Outlook signature in Outlook (it is an anonymized version of the original client's signature, tested to still have the issue):
```adfs fdsdfsa (zij/haar) | asddasfd | asdfasdfasdf | asdfasdfas | asdfasdfasdf | asdfasdf 8, 2223 asdf asdf | asdfasdf 0.18c | asdf 801225, 323 TC asdf | 06 12 12 12 12 | asdf.asdf@asdf.nl | afdsfsdfsdsdfsdf@asdf.nl | www.asdf.nl/asfdasdf/asfdasdfasdf | Aanwezig op ma-di-do-vr```
2. Create a new email
3. Insert the signature
4. Start writing text quickly, pressing enter multiple time by doing so. In our internal testing we just spammed the keyboard
5. If the issue occurs, Outlook will freeze for up to multiple seconds, not inserting the text you are typing until it un-freezes and inserts all the text typed at once

### Actual behavior:
Outlook freezes, up to multiple seconds, seemingly "buffering" the input, before inserting it all at once when the freeze ends

### Expected behavior:
Being able to type text and enter newlines in Outlook without it freezing

### NVDA logs, crash dumps and other attachments:

[nvda-outlook-signature.log](https://github.com/user-attachments/files/20205706/nvda-outlook-signature.log)

#### NVDA installed/portable/running from source:
NVDA installed

#### NVDA version:
NVDA 2024.4.0.34423

#### Windows version:
Microsoft Windows 11 Pro 10.0.26100 Build 26100

#### Name and version of other software in use when reproducing the issue:
Outlook Microsoft 365 MSO (Versie 2504 Build 16.0.18730.20122) 64 bits

#### Other information about your system:
Synthesizer: None selected

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
We have tried using the NVDA 2025.1Beta1. It had the same issue

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested/not applicable

**Comment by seanbudd** [comment]
Can you still reproduce this with NVDA 2025.1.2?

**Comment by Nerlant** [comment]
> Can you still reproduce this with NVDA 2025.1.2?

Yes, I can confirm this is still an issue in NVDA 2025.1.2.
